_test/shared/matlab_xunit_ISISextras/@TestCaseWithSave/TestCaseWithSave.m

```diff
@@ -641,6 +641,8 @@ function assertMethodWithSave (this, var, var_name, funcHandle, varargin)
                         strncmpi(cont{end},'test',4)
                     test_name = cont{end};
                     break
+                else
+                    test_name = cont{end};
```

Comment:
The `if` and `else` block perform the same action. Is that check required?

Reply:
Looking again the `break` is different in two branches, so it can be rewritten with just that in the `if`
```
test_name = cont{end};
if strcmp && ~strcmp && strncmpi
  break;
end
```

Reply:
yes -- this is better